Sims1WidescreenPatcher.UI/Views/MainWindow.axaml

```diff
@@ -6,93 +6,113 @@
-                    <Label>Resolution</Label>
-                    <ComboBox Margin="0 0 5 0"
-                              x:Name="ResolutionCombo"
-                              Items="{Binding Resolutions}"
-                              SelectedItem="{Binding SelectedResolution}"
-                              HorizontalAlignment="Stretch">
-                        <ToolTip.Tip>
-                            <TextBlock Text="Resolutions supported by your monitor.&#x0a;&#x0a;800x600 and 1024x768 are the two default resolutions for The Sims.&#x0a;&#x0a;1600xX resolutions have game breaking graphical issues.">
-                            </TextBlock>
-                        </ToolTip.Tip>
-                    </ComboBox>
-                </StackPanel>
-                <StackPanel Grid.Column="1">
-                    <Label>Wrapper</Label>
-                    <ComboBox Margin="0 0 5 0"
-                              x:Name="WrapperCombo"
-                              Items="{Binding Wrappers}"
-                              SelectedItem="{Binding SelectedWrapper}"
-                              HorizontalAlignment="Stretch">
-                        <ToolTip.Tip>
-                            <TextBlock Text="None: Low framerate. Use if DgVoodoo2 and DDrawCompat don't work.&#x0a;&#x0a;DgVoodoo2: Low/Medium framerate. Use if DDrawCompat doesn't work. (Windows and Linux only)&#x0a;&#x0a;DDrawCompat: High framerate. Use this for borderless fullscreen. Press Shift+F11 for in-game menu. View the wiki for more settings. (Windows only)"></TextBlock>
-                        </ToolTip.Tip>
-                    </ComboBox>
-                </StackPanel>
-                <Button Margin="0 0 5 0"
-                        Grid.Column="2"
-                        x:Name="UninstallButton"
-                        Command="{Binding UninstallCommand}"
-                        Content="Uninstall"
-                        VerticalAlignment="Bottom"
-                        HorizontalAlignment="Stretch"
-                        HorizontalContentAlignment="Center" />
-                <Button Grid.Column="3"
-                        x:Name="PatchButton"
-                        Command="{Binding PatchCommand}"
-                        Content="Patch"
-                        VerticalAlignment="Bottom"
-                        HorizontalAlignment="Stretch"
-                        HorizontalContentAlignment="Center" />
-            </Grid>
-            <!--Row 3-->
-            <Grid Margin="0 5 0 0">
-                <StackPanel>
-                    <Label Padding="2">Progress</Label>
-                    <ProgressBar x:Name="ProgressBar"
-                                 Minimum="0"
-                                 Maximum="100"
-                                 ShowProgressText="True"
-                                 Value="{Binding Progress}" />
-                </StackPanel>
-            </Grid>
-        </StackPanel>
-    </Grid>
+	<Design.DataContext>
```

Comment:
lines 15-66 are unchanged, they are just moved down - real changes are 67 and beyond